github/repos.go

```diff
@@ -284,6 +287,7 @@ func (s *RepositoriesService) Get(owner, repo string) (*Repository, *Response, e
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
 	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	req.Header.Add("Accept", mediaTypeSquashPreview)
```

Comment:
Yeah, pretty sure that 2 accept headers don't work:

```bash
$ curl https://api.github.com/users/technoweenie -I -s -H "Accept: application/vnd.github.drax-preview+json" -H "Accept: application/vnd.github.polaris-preview+json" | grep X-GitHub-Media-Type
X-GitHub-Media-Type: github.drax-preview; format=json
```

Only one seems to have an effect.

I'm not sure if it's even possible to opt into two preview APIs in the same request... Any thoughts on this?